staging/src/k8s.io/apimachinery/pkg/runtime/serializer/json/json.go

```diff
@@ -131,7 +131,7 @@ func gvkWithDefaults(actual, defaultGVK schema.GroupVersionKind) schema.GroupVer
 // If into is provided and the original data is not fully qualified with kind/version/group, the type of the into will be used to alter the returned gvk.
 // If into is nil or data's gvk different from into's gvk, it will generate a new Object with ObjectCreater.New(gvk)
 // On success or most errors, the method will return the calculated schema kind.
-// The gvk calculate priority will be originalData > default gvk > into
+// The gvk calculate priority will be into > default gvk > originalData
```

Comment:
originalData is used to generate: "actual, err := s.meta.Interpret(data)", 
then:		*actual = gvkWithDefaults(*actual, *gvk)

so originalData priority > default gvk > into

Reply:
That's right, I made a mistake, Thanks for your review. Will close.